plugins/woocommerce-admin/client/products/fills/shipping-section/shipping-section-fills.tsx

```diff
@@ -72,107 +81,161 @@ const ShippingSection = () => {
 
 	return (
 		<>
-			<WooProductSectionItem
+			<WooProductSectionItem< ShippingSectionFillPropsType >
 				id={ SHIPPING_SECTION_BASIC_ID }
-				location={ TAB_SHIPPING_ID }
+				tabs={ [
+					{
+						name: TAB_SHIPPING_ID,
+						order: 1,
+						fillProps: {
+							basicSlot: (
+								<WooProductFieldItem.Slot
+									section={ SHIPPING_SECTION_BASIC_ID }
+								/>
+							),
+							dimensionsSlot: (
+								<WooProductFieldItem.Slot
+									section={ SHIPPING_SECTION_DIMENSIONS_ID }
+									fillProps={
+										{
+											setHighlightSide,
+											dimensionProps,
+										} as ShippingDimensionsPropsType
+									}
+								/>
+							),
+						},
+					},
+					{
+						name: VARIANT_TAB_SHIPPING_ID,
```

Comment:
And then here the variant version injects the same slots, but with differing IDs.